docs/includes/guides/migration_3.x.adoc

```diff
@@ -0,0 +1,63 @@
+///////////////////////////////////////////////////////////////////////////////
+
+    Copyright (c) 2022 Oracle and/or its affiliates.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+///////////////////////////////////////////////////////////////////////////////
+
+ifndef::rootdir[:rootdir: {docdir}/../..]
+
+== Java 17 Runtime
+
+Java 11 is no longer supported in Helidon 3. Java 17 or newer is required.
+
+
+== javax.* workspace to jakarta.* workspace
+
+Due to trademark restrictions imposed on the javax.* namespace, Jakarta EE will rename all of its specifications to use jakarta.* in order to move forward with evolving the platform with features as the cloud-native Java platform.
+In version 3.x Helidon supports MicroProfile 5 specification, which now is fully migrated to `jakarta.x` namespace.
+
+We have moved from dependencies in groupId `javax` (Java EE modules) to dependencies
```

Comment:
No we haven't - we have used jakarta groupId even in previous version of Helidon.

Reply:
fixed